Source/WebCore/css/SelectorCheckerTestFunctions.h

```diff
@@ -425,6 +425,14 @@ ALWAYS_INLINE bool matchesFullScreenAnimatingFullScreenTransitionPseudoClass(con
     return element.document().fullscreenManager().isAnimatingFullscreen();
 }
 
+/* FIXME: Remove this temporary hack */
```

Comment:
This should reference a bugzilla bug (and probably say why instead of calling it a hack)